index.d.ts

```diff
@@ -26,7 +26,8 @@ export interface Column {
   lookup?: object;
   render?: (data:any) => any;
   title: string;
-  type?: 'boolean' | 'numeric' | 'date' | 'datetime' | 'time';
+  type?: 'boolean' | 'numeric' | 'date' | 'datetime' | 'time' | 'currency';
+  currencySetting?:object;
```

Comment:
Instead of object you should write something like {x: number, y: string.........} to make currencySetting pattern more clear

Reply:
Ok I'll refactor my code as you mentioned above